### ISSUE TITLE
linter: import/named throwing error on named typescript enum import into javascript

### ISSUE THREAD

**Issue by MichaelGoff** [oxc-project/oxc#9601]
### What version of Oxlint are you using?

0.15.13

### What command did you run?

`oxlint`

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "env": {
    "browser": true
  },
  "ignorePatterns" : ["node_modules"],
  "categories": {
    "correctness": "error"
  },
  "plugins": [
    "import",
    "typescript"
  ],
  "rules": {
    "import/named": "error"
  }
}
```

### What happened?

Oxlint reports an error from `import/named` when importing a `ts` enum into a `.js` file. eslint-plugin-import does not report this as an issue. 

Given `enums.ts`
```ts
export enum Events {
    Christmas = 'Christmas',
    NewYears = 'NewYears',
}
```

and `jsFile.js`
```js
import {Events} from "./enums";

console.log(Events.NewYears);
```

Oxlint will report the following false positive error:

```
  × eslint-plugin-import(named): named import "Events" not found
   ╭─[jsFile.js:1:9]
 1 │ import {Events} from "./enums";
   ·         ──────
 2 │
   ╰────
  help: does "./enums" have the export "Events"?
```

It does not report such an error when importing an enum into a `.ts` file. 

See reproduction Repo: https://github.com/MichaelGoff/oxlint-named-enum-js-bug

**Comment by camc314** [comment]
i can't reproduce this on the latest version anymore. i think this has been fixed in https://github.com/oxc-project/oxc/pull/9383

@MichaelGoff do you mind trying it again on the latest version? 🙏

**Comment by MichaelGoff** [comment]
I can confirm as of Oxlint 1.4.0 I can no longer reproduce this issue. Thanks!
